test/tests/infrastructure/RequestHelper.js

```diff
@@ -1,4 +1,7 @@
 import { RequestHelper } from '../../../src/infrastructure';
+import Gitlab from '../../..';
```

Comment:
Please fix the import statement: https://travis-ci.org/jdalrymple/node-gitlab/jobs/419127291#L3209-L3210